exercises/concept/census/census.go

```diff
@@ -1,4 +1,4 @@
-// Packages census simulates a system used to collect census data.
+// Package census simulates a system used to collect census data.
```

Comment:
Could you apply the same fix for the .meta/exemplar.go file please?

Reply:
Done! Sorry for missing that.